### ISSUE TITLE
Empty autoprefixer.yml breaks my rake task

### ISSUE THREAD

**Issue by sanemat** [ai/autoprefixer-rails#123]
I have empty `autoprefixer.yml`, and this breaks my rake task after this commit:
https://github.com/ai/autoprefixer-rails/commit/abefbe4643da89f1928c665f68de2b0ec2b66d4a

I got this error:
```
rake aborted!
NoMethodError: undefined method `symbolize_keys' for false:FalseClass
/path/to/vendor/bundle/ruby/2.3.0/gems/autoprefixer-rails-7.1.2.1/lib/autoprefixer-rails/railtie.rb:37:in `config'
(snip)
```

Because `::YAML.load("")` returns `false` and `::YAML.load_file(empty_file)` returns `false`.

```ruby
::YAML.load("") #=> false
```
Is this expected behavior?

**Comment by ai** [comment]
Nope.

Do you want make PR to fix it to save your name in project history?

**Comment by sanemat** [comment]
I make PR #124 :smile_cat:

**Comment by ai** [comment]
Thanks! Released in 7.1.2.2.
